packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -0,0 +1,82 @@
+import axios from "axios";
+import { MetriportError } from "../../../util/error/metriport-error";
+import { S3Utils, parseS3FileName } from "../s3";
+
+const api = axios.create();
+
+const UPLOADS_FOLDER = "uploads";
+
+export type FileData = {
+  mimeType?: string | undefined;
+  size?: number | undefined;
+  originalName: string;
+  locationUrl: string;
+  docId: string;
+};
+
+export async function documentUploaderHandler(
+  sourceBucket: string,
+  sourceKey: string,
+  destinationBucket: string,
+  region: string,
+  apiServerURL: string
+) {
+  const s3Utils = new S3Utils(region);
+  const copySource = encodeURI(`${sourceBucket}/${sourceKey}`);
+  const s3FileNameParts = parseS3FileName(sourceKey);
+  if (!s3FileNameParts) {
+    const message = "Failed to parse S3 file key";
+    console.log(message);
+    throw new MetriportError(message, null, { sourceBucket, sourceKey });
+  }
+  const { cxId, patientId, docId } = s3FileNameParts;
+  const destinationKey = buildDestinationKey(cxId, patientId, docId);
+
+  const params = {
+    CopySource: copySource,
+    Bucket: destinationBucket,
+    Key: destinationKey,
+  };
+
+  // Make a copy of the file to the general medical documents bucket
+  try {
+    await s3Utils.s3.copyObject(params).promise();
+  } catch (error) {
+    const message = "Error copying the uploaded file to medical documents bucket";
+    console.log(message, JSON.stringify(error));
```

Comment:
This doesn't always result in what one would expect, that why I ended up building the `errorToString()` function - [link](https://github.com/metriport/metriport/blob/b603f9f7f26963ca00df2abc1a59c1c3801bb1ad/packages/api/src/shared/log.ts#L20).

Its not on core yet, maybe we can add it there, or just log `${error}`?

Reply:
😬 i'll just use `${error}` for now if that's ok